### ISSUE TITLE
Add BeautifulSoup and Selenium

### ISSUE THREAD

**Issue by langdon** [BU-Spark/bu-spark#43]
We should add BeautifulSoup and Selenium to the data science tech stack as a scraping recommendation.

**Comment by ayang903** [comment]
@langdon I'm working on putting together this guide, do you have any recommendations for APIs or websites that could serve as practical examples for the guide?

**Comment by langdon** [comment]
not really, basically just googling and the docs. I did give https://realpython.com/beautiful-soup-web-scraper-python/ to one of my classes as suggestion.
